addons/keybinding/fnc_addKeybind.sqf

```diff
@@ -88,6 +88,16 @@ if (_defaultKeybind isEqualTypeParams [0, false, false, false]) then {
     _defaultKeybind = [_defaultKey, [_defaultShift, _defaultControl, _defaultAlt]];
 };
 
+// Handle non-code type being passed for up/down code args
+if (!(_downCode isEqualType {})) then {
+    if (!(_downCode isEqualTo "")) then {WARNING_1("_downCode: Non code type [%1] will be ignored",_downCode);};
+    _downCode = {};
+};
+if (!(_upCode isEqualType {})) then {
+    if (!(_upCode isEqualTo "")) then {WARNING_1("_upCode: Non code type [%1] will be ignored",_upCode);};
+    _upCode = {};
+};
```

Comment:
`params` already guarantees that the type is either CODE or STRING, so `!( isEqualType {})` and `!( isEqualType "")` is tautologous.
I really don't see whats wrong with just compiling the "code" if it's STRING type.